devon4j.properties

```diff
@@ -0,0 +1,14 @@
+category=Governance
+description=Plugin for SonarQube to validate devon4j architecture and conventions.
+homepageUrl=https://github.com/devonfw/sonar-devon4j-plugin/#sonar-devon4j-plugin
+archivedVersions=
+publicVersions=3.2.1
+
+defaults.mavenGroupId=com.devonfw.tools
+defaults.mavenArtifactId=sonar-devon4j-plugin
+
+3.2.1.description=First official release under the name sonar-devon4j-plugin and initial entry into the marketplace.
+3.2.1.sqVersions=[6.4,7.*]
```

Comment:
This range needs to be updated to reflect current SQ versions/support. Assuming compatibility, I suggest: `[7.9,LATEST]`
Note that only one version at a time can be marked compatible with latest, so on your second release you'll need to update this version. But using `LATEST` has the benefit that as new versions of SQ are released, your most recent plugin version is automatically marked compatible.